torch/nn/modules/_functions.py

```diff
@@ -7,7 +7,12 @@ class SyncBatchNorm(Function):
 
     @staticmethod
     def forward(self, input, weight, bias, running_mean, running_var, eps, momentum, process_group, world_size):
-        input = input.contiguous()
+        if not input.is_contiguous(memory_format=torch.channels_last):
+            input = input.contiguous()
+
+        size = int(input.numel() / input.size(1))
```

Comment:
is there a check anywhere that input.size(1) is not 0? `input.numel() // input.size(1)`

Reply:
I added a check in python here https://github.com/pytorch/pytorch/pull/46906/commits/d68394e0dd96d4db2f374e5787e279f13bb9fcc9